src/UserService.Data.Provider.MsSql.Ef/Migrations/20210317164900_AddDbSkills.cs

```diff
@@ -0,0 +1,71 @@
+﻿using LT.DigitalOffice.UserService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(UserServiceDbContext))]
+    [Migration("20210317164900_AddDbSkills")]
+    class AddDbSkills : Migration
+    {
+        private void CreateSkillsTable(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: DbSkill.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    SkillName = table.Column<string>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Skill", x => x.Id);
```

Comment:
if your table calls "Skills" I think, you should call PrimaryKey "PK_Skills"

Reply:
fixed